### PR TITLE
add clinic & slots

### PR DESCRIPTION
**Pull request by saeedkhal**

### CODE REVIEW

File: models/clinicModel.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 const mongoose = require('mongoose');
 
 const clinicSchema = new mongoose.Schema({
-  doctor: {
-    type: [mongoose.Schema.Types.ObjectId],
+  doctorId: {
```
Comment (RamadanIbrahem98) [inline models/clinicModel.js:4]: why did you change it back XD
keep it consistent as **doctor**

File: models/clinicModel.js [hunk 2]
```diff
@@ -18,7 +18,7 @@ const clinicSchema = new mongoose.Schema({
     max: 5,
     min: 0,
   },
-  phoneNumber: {
+  phoneNumbers: {
```
Comment (RamadanIbrahem98) [inline models/clinicModel.js:21]: Here in the clinic, it's good to add multiple numbers, but we need to extend the validators to include landline numbers

File: models/slotModel.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+const mongoose = require('mongoose');
```
Comment (RamadanIbrahem98) [inline models/slotModel.js:1]: why did you repeat this file?
@HamdyMohamed10 has already written it.

and hence, the validator file should also be edited

Comment (saeedkhal) [reply models/slotModel.js:1]: @HamdyMohamed10  slot model have been updated  so that i forgot to delete after add my new one 
i will add fix it now

File: models/clinicModel.js (hunk 2 above)
Comment (saeedkhal) [reply models/clinicModel.js:21]: OK i will fix Right now

File: models/clinicModel.js (hunk 1 above)
Comment (saeedkhal) [reply models/clinicModel.js:4]: ok if we do that we have a problem right now so when doctor create a new  clinic how this doctor will add new doctor , as long as user will book with one doctor not at group of doctors , so solution is to  keep it for one doctor .

Comment (saeedkhal) [reply models/clinicModel.js:4]: i think it depend on the future  work  if we populate the id or not, so let leave it as it is now

File: routes/doctorRouter.js [hunk 4]
```diff
@@ -55,4 +57,27 @@ router
     ),
     doctorController.updateDoctorProfile
   );
+router.use('/clinics', clinicRouter);
+
+// router
```
Comment (RamadanIbrahem98) [inline routes/doctorRouter.js:62]: don't leave out comments 😅
delete them

**Review by RamadanIbrahem98 — APPROVED** [review]

File: routes/doctorRouter.js (hunk 4 above)
Comment (RamadanIbrahem98) [reply routes/doctorRouter.js:62]: After committing merge and delete the branch
